pkg/operator/ceph/file/mds/mds.go

```diff
@@ -101,6 +102,18 @@ func (c *Cluster) Start() error {
 		return errors.Wrap(err, "error checking pod memory")
 	}
 
+	// upgrading MDS cluster needs to set max_mds to 1 and stop all stand-by MDSes first
+	isUpgrade, err := c.isCephUpgrade()
+	if err != nil {
+		return errors.Wrapf(err, "failed to determine if MDS cluster for filesystem %s needs upgraded", c.fs.Name)
+	}
+	if isUpgrade {
+		if err := c.upgradeMDS(); err != nil {
+			return errors.Wrapf(err, "failed to upgrade MDS cluster for filesystem %s", c.fs.Name)
+		}
+		logger.Infof("successfully ugraded MDS cluster for filesystem %s", c.fs.Name)
```

Comment:
Three things:
1. This prints before the non-active daemons are updated, so the upgrade cannot **quite** be considered complete.
2. Missed a `p` in "upgraded"
3. We use %q to quote the names of resources in logs for clarity

```suggestion
		logger.Infof("successfully upgraded primary MDS for filesystem %q. other MDSes will now be upgraded", c.fs.Name)
```